measurement/util.py

```diff
@@ -146,3 +164,7 @@ def get_hostname():
     # This function is merely used, so only import socket package when necessary
     import socket
     return socket.gethostname()
+
+def python_version():
+    # get a numeric Python version
+    return sys.version_info[0] + sys.version_info[1] / 10
```

Comment:
major and minor are integers.
>>> sys.version_info
sys.version_info(major=2, minor=7, micro=10, releaselevel='final', serial=0)
>>> sys.version_info[0] + sys.version_info[1] / 10
2

Reply:
OK, I'll change to `sys.version_info[0] + sys.version_info[1] * 0.1`